pipelines/_ingredients/README.md

```diff
@@ -0,0 +1,9 @@
+# Pipeline Definition Ingredients
+
+This directory contains ingredient pipeline definition YAML files.
+The pipelines defined here are intended to be used as the basic blocks
+for calibration verification.  As such, they are not intended to be
+used directly by end-users for science purposes.
+
+Science ready pipeline definitions for specific cameras are located in
+the [pipelines](../pipelines) directory.
```

Comment:
I think this just needs to be `[pipelines](..)` to link properly. Whilst you're formatting, would it be possible to have one sentence per line?